### PR TITLE
Update lighting effects

### PR DESCRIPTION
**Pull request by XBowsTech**
## Description
Update lighting effects

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by XBowsTech** [comment]
Hey qmk/qmk_firmware

Thank you for contacting us. This is an automated message to let you know that we have received your message. We will get back to you as soon as possible.

Best,
The x-bows team

File: keyboards/xbows/ranger/rules.mk [hunk 1]
```diff
@@ -13,7 +13,7 @@ EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
-SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/xbows/ranger/rules.mk:16]: Any reason for this?

**Comment by XBowsTech** [comment]
Hey qmk/qmk_firmware

Thank you for contacting us. This is an automated message to let you know that we have received your message. We will get back to you as soon as possible.

Best,
The x-bows team

File: keyboards/xbows/ranger/rules.mk (hunk 1 above)
Comment (XBowsTech) [reply keyboards/xbows/ranger/rules.mk:16]: My keyboard always wakes up when my computer is asleep, this seems to be one way, and I'm looking for the optimal solution.

Comment (fauxpark) [reply keyboards/xbows/ranger/rules.mk:16]: I don't think this feature does what you think it does. It essentially pulses the Caps Lock LED when the host PC is in sleep mode, however this board has no Caps Lock LED defined, so it won't do anything.

**Comment by XBowsTech** [comment]
Hey qmk/qmk_firmware

Thank you for contacting us. This is an automated message to let you know that we have received your message. We will get back to you as soon as possible.

Best,
The x-bows team

File: keyboards/xbows/ranger/rules.mk (hunk 1 above)
Comment (XBowsTech) [reply keyboards/xbows/ranger/rules.mk]: OK

**Comment by XBowsTech** [comment]
Hi fauxpark,
I modified "SLEEP_LED_ENABLE = no" yesterday and was reminded that there was rules.mk conflict. Do I need to submit the merger application again?
@fauxpark

**Comment by fauxpark** [comment]
No, you can just press the "resolve conflicts" button above and remove the entire conflicting section (the two SLEEP_LED_ENABLE lines and the markers), as SLEEP_LED_ENABLE no longer needs to be in rules.mk.

**Comment by XBowsTech** [comment]
Hey qmk/qmk_firmware

Thank you for contacting us. This is an automated message to let you know that we have received your message. We will get back to you as soon as possible.

Best,
The x-bows team

**Comment by XBowsTech** [comment]
@fauxpark  Thanks to you, I have solved the conflict problem with your help.

File: keyboards/xbows/ranger/rules.mk [hunk 2]
```diff
@@ -12,6 +12,7 @@ MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
+
```
Comment (fauxpark) [inline keyboards/xbows/ranger/rules.mk:15]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by XBowsTech** [comment]
Hey qmk/qmk_firmware

Thank you for contacting us. This is an automated message to let you know that we have received your message. We will get back to you as soon as possible.

Best,
The x-bows team

**Comment by XBowsTech** [comment]
@fauxpark   Hi fauxpark, Can you merge it for me?

**Comment by XBowsTech** [comment]
@fauxpark @drashna  Can you merge it for me? Thanks！
If there is anything I need to change, please let me know.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**